truss/server/control/server.py

```diff
@@ -72,8 +59,8 @@ def run(self):
 
 if __name__ == "__main__":
     control_server = ControlServer(
-        python_executable_path=_identify_python_executable_path(),
-        inf_serv_home=os.environ["APP_HOME"],
+        python_executable_path=os.environ.get("PYTHON_EXECUTABLE", default="python3"),
```

Comment:
we're using environment variables for a lot of options now. One of the downsides of this is that there is no single place to see what available options are. I think a reasonable refactor for now would be to have a settings file that has all of the `os.environ.get` calls, and then these other code paths just import the constant from there.